### ISSUE TITLE
[🚀 Feature]: deployments should offer parameters for configuring the securityContext 

### ISSUE THREAD

**Issue by manuschillerdev** [SeleniumHQ/docker-selenium#1825]
### Feature and motivation

It is best practice to have containers run as root with the least possible privileges.
It would be convenient if the deployments offered the possibility to configure `securityContext` via the `values.yaml` file.

### Usage example

```values.yaml
selenium-grid:
  hub:
    securityContext:
      runAsNonRoot: true
      runAsUser: 1001 
```

**Comment by diemol** [comment]
@manuschillerdev would you like to send a PR to improve this?

**Comment by diemol** [comment]
Closed via #1845
